app/controllers/index.erb

```diff
@@ -0,0 +1,8 @@
+<h1>Welcome to our news aggregator!</h1>
```

Comment:
hahahaha how did I end up in News Aggregator??? I'm scared! 

(Definitely delete this file -- it's also a view in a controller dir)